src/constants/widths.js

```diff
@@ -0,0 +1,6 @@
+export default {
```

Comment:
Name could be more specific as to what widths?

Reply:
Can you elaborate this point please? I have currently carried forward the old `quill` widths for backward compatibility, but we can always update it.

Reply:
The name of the file could more specific than just widths is the suggestion.

Reply:
Ah, right, that makes sense. How about `HtmlElementWidths`?

Reply:
htmlElementWidths?

Reply:
Haha, my bad with the casing 🙇 ! Will go ahead with `htmlElementWidths`

Reply:
Done: c5067af